.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,25 @@
+---
+name: "\U0001F680 Feature request"
+about: Suggest a new feature or a big change
+
+---
+<!-- Thank you for contributing. These HTML commments will not render in the issue, but you can delete them once you've read them if you prefer! -->
+
+### Proposed change
+<!-- Use this section to describe the feature you'd like to be added. -->
+
+
+### Alternative options
+<!-- Use this section to describe alternative options and why you've decided on the proposed feature above. -->
+
+
+### Who would use this feature?
+<!-- Describe the audience for this feature. This information will affect who chooses to work on the feature with you. -->
```

Comment:
```suggestion
<!-- Describe who would benefit from using this feature. -->
```

Reply:
I've considered part `who chooses to work on the feature with you.`, it implies it will be a collaboration and the person suggesting the feature will be involved doing so.

I like the idea of having these as slimmed as possible, and I think it has some value to communicate the value information could have, but we cannot communicate everything so I figured perhaps we should slim this down a bit.